src/anybody.js

```diff
@@ -444,7 +444,7 @@ export class Anybody extends EventEmitter {
         velocity: this.createVector(0, 0),
         radius,
         starLvl: 0,
-        maxStarLvl: 3,
+        maxStarLvl: Math.random() > 0.2 ? 1 : 3,
```

Comment:
minimum `maxStarLvl` is currently 3

```suggestion
        maxStarLvl: Math.random() > 0.2 ? 3 : 5,
```

Reply:
👍 i did this to test the withering animation on first round of play